examples/official-storybook/config-addon-jest.json

```diff
@@ -1,28 +0,0 @@
-{
-  "verbose": true,
-  "coverageDirectory": "coverage/",
-  "notify": true,
-  "transform": {
-    ".*": "babel-jest"
-  },
-  "snapshotSerializers": [
-    "enzyme-to-json/serializer"
-  ],
-  "setupFiles": [
-    "<rootDir>/src/enzyme.js"
-  ],
-  "moduleNameMapper": {
-    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
-    "\\.s?css$": "<rootDir>/__mocks__/styleMock.js"
-  },
-  "collectCoverage": true,
-  "collectCoverageFrom": [
-    "**/*.js",
-    "*.js",
-    "!.*",
-    "!__test__/*",
-    "!**/__test__/*",
-    "!**/index.js",
-    "!**/story.js"
-  ]
-}
```

Comment:
Why is this file removed?

Reply:
Because it is not used there :) This file was used only inside the `cra-kitchen-sink` example with addon-jest.

Reply:
Oh, so you moved `addon-jest` example from `official-storybook` back to `cra-kitchen-sink`

The idea of #2504 is to make kitchen-sink examples tiny and move all the documenting stories to `official-storybook`

Reply:
Please revert that move as well